CITATION.cff

```diff
@@ -0,0 +1,45 @@
+cff-version: 1.2.0
+title: DLA-Future-Fortran 0.1.0
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: John
+    family-names: Biddiscombe
+    orcid: 'https://orcid.org/0000-0002-6552-2833'
+  - given-names: Alberto
+    family-names: Invernizzi
+    orcid: 'https://orcid.org/0009-0004-4948-0313'
+  - given-names: Rocco
+    family-names: Meli
+    orcid: 'https://orcid.org/0000-0002-2845-3410'
+  - given-names: Auriane
+    family-names: Reverdell
+    orcid: 'https://orcid.org/0000-0002-5531-0458'
+  - given-names: Mikael
+    family-names: Simberg
+    orcid: 'https://orcid.org/0000-0002-7238-8935'
+  - given-names: Raffaele
+    family-names: Solcà
+    orcid: 'https://orcid.org/0009-0009-9346-4376'
+  - given-names: Mathieu  
+    family-names: Taillefumier
+    orcid: 'https://orcid.org/0000-0002-3983-5625'
+identifiers:
+  - type: doi
+    value:
```

Comment:
I'm not entirely sure what should go here before doing an actual release.

Reply:
I suspect that this has to be left out from the first release. The field is optional anyway, and then you could fill in the concept DOI (and/or the versioned DOI) after 0.1.0 is out? As far as I understand it this field won't make a difference for the zenodo release anyway, but I may be wrong.

Reply:
https://github.com/eth-cscs/DLA-Future-Fortran/issues/4